nginx/nginx.conf

```
@@ -0,0 +1,45 @@
+user nginx;
+worker_processes 4;
+
+error_log /var/log/nginx/error.log warn;
+pid /var/run/nginx.pid;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  gzip on;
+
+  keepalive_timeout 65;
+
+  server {
+    listen 9000;
```

Ez direkt lett 9000, vagy véletlen maradt ez?
```suggestion
    listen 80;
```

---

Lehet 80 is, de igazabol ugyis majd a docker hoston kell megfeleloen portolni, es kiengedni.